.github/workflows/run-tests.yaml

```diff
@@ -0,0 +1,26 @@
+name: Run Tests
+on:
+  pull_request:
+    types: [opened, reopened]
```

Comment:
This will only run when the pull request is opened, not pushed to. We at least want to add `synchronize` to the types and could just run it `on: [push]`